src/Components/Query.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import '../styles/query.css';
+
+const Query = ({ handleChange }) => (
+  <form className="form">
+    <span className="search-by" />
+    <input className="input-search" type="text" placeholder="Enter country name ..." onChange={handleChange} />
+  </form>
+);
```

Comment:
- [ ] Looking at the image below, your search box does not display properly. The **world_map** element covers a part of your **input-search** element. Users have to move to the right in order to access the input. Kindly fix this so that on hovering over any part of the search box users can access it.

![redwing-search](https://user-images.githubusercontent.com/36542986/142622087-7dbcdb1b-f068-49b9-b468-8e7d07551415.jpg)